src/main/java/org/jenkinsci/plugins/mesos/MesosApi.java

```diff
@@ -2,10 +2,7 @@
 
 import akka.NotUsed;
 import akka.actor.ActorSystem;
-import akka.stream.ActorMaterializer;
-import akka.stream.KillSwitch;
-import akka.stream.KillSwitches;
-import akka.stream.OverflowStrategy;
+import akka.stream.*;
```

Comment:
we do NOT want to collapse to `*`

Reply:
We are using the Google Java Format with Spottless. We will follow up an modify the format to avoid star imports.